docs/v1.md

```diff
@@ -226,7 +251,7 @@
 - servers() : `Server[]`
 - summary() : `string` | `undefined`
 - tags() : `Tags`
-- security() : `Array<Record<string, { schema: SecurityScheme, scopes: Array<string> }>>`
+- security() : `SecurityRequirements`
```

Comment:
I have problem here. `SecurityRequirements` in this PR is a list/array of singles `SecurityRequirement` which has only two methods `scheme` and `scopes` and here is a problem. In the spec `Sec` is a map of pointed schemas and scopes -> https://github.com/asyncapi/spec/blob/master/spec/asyncapi.md#security-requirement-object so you can have multiple schemas in single security "item", like this one:

```yaml
security:
  - apiKeyScheme: []
     oauthScheme: ['...']
  - openIDScheme: []
``` 

Above example means that you wanna be send/receive something to/given channel you need be authorize against `apiKeyScheme` and `oauthScheme` or only `openIDScheme`. I don't know how we should change it, to the `Array<SecurityRequirements>`? Wdyt?

Reply:
Yeah, I know, it's hard to "conduct", but in spec you have array of object (of security schemes), not array of security scheme (single security scheme). If it's an object with keys then from the point of view of the spec's syntax itself you can use 2 schemas in one requirement.

cc @fmvilas

Reply:
Ok, now i got it (thanks @magicmatatjahu). Taking the definition of `security` field of `servers` (for example), it says it is an array of Security Requirement. Indeed it feels weird to me, but in the meantime I will change then this to become `SecurityRequirements[]`, so we are flexible enough to support the current behavior but also any future.